ownCloudAppShared/Client/User Interface/SortBar.swift

```diff
@@ -56,7 +88,7 @@ public class SortBar: UIView, Themeable, UIPopoverPresentationControllerDelegate
 	}
 
 	// MARK: - Constants
-	let sideButtonsSize: CGSize = CGSize(width: 44.0, height: 44.0)
+	let sideButtonsSize: CGSize = CGSize(width: 22.0, height: 22.0)
```

Comment:
I thought we set this to 44.0 before, because it was not so good to tap, because of the small size before. Please also have a look, if it is aligned with the `…` of the more button in the file list cell. Before the UI looked a little bit "jumpy".

Reply:
That was also my worry initially, but it didn't seem to be an issue in tests when I tried. I chose a smaller size so that the "Reveal" arrow also fits in without excessive white space between the "more dots" and the "reveal" arrow.

In my testing, the dots still aligned well between navigation bar and table view rows:

More | More + Reveal
--|--
<img width="568" alt="Bildschirmfoto 2021-04-08 um 22 59 56" src="https://user-images.githubusercontent.com/639669/114095781-252f5800-98be-11eb-90b5-9bb381f41317.png"> | <img width="568" alt="Bildschirmfoto 2021-04-08 um 23 00 26" src="https://user-images.githubusercontent.com/639669/114095839-3710fb00-98be-11eb-9808-4fe2cab2df5d.png">

Reply:
The problem is on a real device. On simulator everything works fine, but on the iPhone it is difficult to touch.